app/models/user.rb

```diff
@@ -15,6 +15,12 @@ class User < ActiveRecord::Base
     authentication_keys: [:email]
   )
 
+  include EncryptableAttribute
+
+  encrypted_attribute(name: :phone, default: :nil)
+  encrypted_attribute(name: :otp_secret_key, default: :nil)
+  encrypted_attribute(name: :email, default: "''", setter: false)
```

Comment:
the default value seems very very strange to me, because we have to worry about quoting schemes?

I have an idea, we could do some type checking to make sure the default value provided is either `nil`, `Symbol` or `String` and then call `#inspect` on it to make method body for the eval